cmd/describe/addon/cmd_test.go

```diff
@@ -0,0 +1,53 @@
+package addon
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/format"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/spf13/cobra"
+)
+
+func TestDescribeAddon(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "rosa describe addon")
+}
+
+var _ = Describe("describe addon", func() {
+	Context("describe addon command", func() {
+		It("returns command", func() {
+			cmd := NewDescribeAddonCommand()
+			Expect(cmd).NotTo(BeNil())
+		})
+	})
+
+	Context("execute describe addon", func() {
```

Comment:
AFAICT, the command isnt executed in these tests, only creation of the command, and the use of the args function from cobra